imod/logging/logging_decorators.py

```diff
@@ -0,0 +1,34 @@
+from imod.logging import logger
+from imod.logging.loglevel import LogLevel
+
+
+# decorator to print log messages announcing the begin and end of the decorated method
+def standard_log_decorator(start_level: LogLevel = LogLevel.INFO, end_level: LogLevel = LogLevel.DEBUG):
+    def decorator(fun):
+        def wrapper(*args, **kwargs):
+            object_type = str(type(args[0]))
```

Comment:
This prints nicer strings, e.g. ``Drainage`` instead of ``<class 'imod.mf6.drn.Drainage'>``:

```suggestion
            object_name = str(type(args[0]).__name__)
```